docs/general/builtins/scalar-functions.rst

```diff
@@ -2593,6 +2593,32 @@ An alias for :ref:`scalar-array_upper`.
     SELECT 1 row in set (... sec)
 
 
+.. NOTE::
+
+    For compatibility with PostgreSQL, :ref:`array_length <scalar-array_length>`
```

Comment:
Is compatibility with PG the main reason of returning `null` or we are following standard?

Reply:
Thanks. I've removed the amendment to `scalar-functions.rst` completely.